deb/build/debian/jenkins.default

```diff
@@ -20,7 +20,7 @@ JENKINS_USER=$NAME
 JENKINS_GROUP=$NAME
 
 # location of the jenkins war file
-JENKINS_WAR=/usr/share/$NAME/$NAME.war
+JENKINS_WAR=/usr/share/java/$NAME.war
```

Comment:
is this supposed to be `/usr/share/jenkins`?

Reply:
Please read [this](https://wiki.debian.org/Java/Packaging) and [this](https://docs.fedoraproject.org/en-US/packaging-guidelines/Java/) before leaving such comments.

Reply:
If you expect people to read such things at least link them in the description, (which I checked before asking)

Reply:
> If you expect

I have no such expectation, but I do expect that someone asking "is this supposed to be X?" would provide their reasoning.